File: unittest/scripts/auto/js_devapi/scripts/mysqlx_tableselect_help.js
//@ Initialization
shell.connect(__uripwd + '/mysql');
var table = db.user;
var crud = table.select();
session.close();

//@ TableSelect help
crud.help();

//@ TableSelect help, \? [USE:TableSelect help]
\? TableSelect

//@ Help on bind
crud.help('bind');

//@ Help on bind, \? [USE:Help on bind]
\? TableSelect.bind

//@ Help on execute
crud.help('execute');

//@ Help on execute, \? [USE:Help on execute]
\? TableSelect.execute

//@ Help on groupBy
crud.help('groupBy');

//@ Help on groupBy, \? [USE:Help on groupBy]
\? TableSelect.groupBy

//@ Help on having
crud.help('having');

//@ Help on having, \? [USE:Help on having]
\? TableSelect.having

//@ Help on help
crud.help('help');

//@ Help on help, \? [USE:Help on help]
\? TableSelect.help

//@ Help on limit
crud.help('limit');

//@ Help on limit, \? [USE:Help on limit]
\? TableSelect.limit

//@ Help on lockExclusive
crud.help('lockExclusive');

//@ Help on lockExclusive, \? [USE:Help on lockExclusive]
\? TableSelect.lockExclusive

//@ Help on lockShared
crud.help('lockShared');

//@ Help on lockShared, \? [USE:Help on lockShared]
\? TableSelect.lockShared

//@ Help on offset
crud.help('offset');

//@ Help on offset, \? [USE:Help on offset]
\? TableSelect.offset

//@ Help on orderBy
crud.help('orderBy');

//@ Help on orderBy, \? [USE:Help on orderBy]
\? TableSelect.orderBy

//@ Help on select
crud.help('select');

//@ Help on select, \? [USE:Help on select]
\? TableSelect.select

//@ Help on where
crud.help('where');

//@ Help on where, \? [USE:Help on where]
\? TableSelect.where

File: unittest/scripts/auto/py_devapi/scripts/mysqlx_tableselect_help.py
shell.connect(__uripwd + '/mysql')
tableselect = db.user.select()
session.close()

#@ tableselect
tableselect.help()

#@ global ? for TableSelect[USE:tableselect]
\? TableSelect

#@ global help for TableSelect[USE:tableselect]
\help TableSelect

#@ tableselect.bind
tableselect.help('bind')

#@ global ? for bind[USE:tableselect.bind]
\? TableSelect.bind

#@ global help for bind[USE:tableselect.bind]
\help TableSelect.bind

#@ tableselect.execute
tableselect.help('execute')

#@ global ? for execute[USE:tableselect.execute]
\? TableSelect.execute

#@ global help for execute[USE:tableselect.execute]
\help TableSelect.execute

#@ tableselect.group_by
tableselect.help('group_by')

#@ global ? for group_by[USE:tableselect.group_by]
\? TableSelect.group_by

#@ global help for group_by[USE:tableselect.group_by]
\help TableSelect.group_by

#@ tableselect.help
tableselect.help('help')

#@ global ? for help[USE:tableselect.help]
\? TableSelect.help

#@ global help for help[USE:tableselect.help]
\help TableSelect.help

#@ tableselect.limit
tableselect.help('limit')

#@ global ? for limit[USE:tableselect.limit]
\? TableSelect.limit

#@ global help for limit[USE:tableselect.limit]
\help TableSelect.limit

#@ tableselect.lock_exclusive
tableselect.help('lock_exclusive')

#@ global ? for lock_exclusive[USE:tableselect.lock_exclusive]
\? TableSelect.lock_exclusive

#@ global help for lock_exclusive[USE:tableselect.lock_exclusive]
\help TableSelect.lock_exclusive

#@ tableselect.lock_shared
tableselect.help('lock_shared')

#@ global ? for lock_shared[USE:tableselect.lock_shared]
\? TableSelect.lock_shared

#@ global help for lock_shared[USE:tableselect.lock_shared]
\help TableSelect.lock_shared

#@ tableselect.order_by
tableselect.help('order_by')

#@ global ? for order_by[USE:tableselect.order_by]
\? TableSelect.order_by

#@ global help for order_by[USE:tableselect.order_by]
\help TableSelect.order_by

#@ tableselect.where
tableselect.help('where')

#@ global ? for where[USE:tableselect.where]
\? TableSelect.where

#@ global help for where[USE:tableselect.where]
\help TableSelect.where
